scripts/linting/mdn-sitemap-compare.js

```diff
@@ -0,0 +1,130 @@
+/*
+A linter that can check that all of our URLs match URLs in MDN's sitemaps.
+
+Usage:
+
+npm run mdn-sitemap-compare [locale [locale2]]
+
+*/
+
+const url = require("url");
+const fetch = require("node-fetch");
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+
+const readFile = util.promisify(fs.readFile);
+
+const baseURL = "https://developer.mozilla.org";
+
+const ROOT = path.join(__dirname, "..", "..");
+
+async function fetchSitemap(url) {
+    if (url.startsWith("/")) url = `${baseURL}${url}`;
+    let r = await fetch(url);
+    let payload = await r.text();
+    return payload;
+}
+
+async function getLocs(url) {
+    const payload = await fetchSitemap(url);
+    return [...payload.matchAll(/<loc>(.*?)<\/loc>/g)].map(m => m[1]);
```

Comment:
As we said, `matchAll` needs Node 12. But, according to https://nodejs.org/en/about/releases/ , Node 12 will be the LTS in exactly two weeks! So maybe it's OK :).

Reply:
I'm failing to find where we decided to use Node >=10. It's not in the `package.json` as an `engine`. 
Was it just something we chatted about and agreed in verbal only?

I'm actually OK with leaving this code here because it'll definitely work on Travis and the only kind of people who'd use and hack on this (similar to the scraper) are the "hard core" users like yourself. And we should make sure we have modern tools installed.

So I'd be tempted to just leave this comment be for now.

Reply:
We didn't fully resolve it but it was discussed here: https://github.com/mdn/stumptown-content/issues/141. But the LTS is 12 now anyway, so using `matchAll` here is fine :).